apps/builder/app/builder/features/breakpoints/width-input.tsx

```diff
@@ -55,15 +50,21 @@ const useEnhancedInput = ({
     onChangeComplete: handleChangeComplete,
   });
 
+  const getValue = () => {
+    const value = inputRef.current?.valueAsNumber;
+    return typeof value !== "number" || isNaN(value) ? min : value;
```

Comment:
typeof value !== "number" could be undefined check

isNaN -> Number.isNaN

Reply:
good catch